src/encoded/static/components/encyclopedia.js

```diff
@@ -1,49 +1,492 @@
+import React from 'react';
 import PropTypes from 'prop-types';
 import url from 'url';
-import { Panel, PanelBody } from '../libs/ui/panel';
-import { ResultTable } from './search';
+import { Panel, TabPanel } from '../libs/ui/panel';
+import GenomeBrowser from './genome_browser';
 import * as globals from './globals';
+import getSeriesData from './series_search.js';
+import { FacetList } from './search';
+import { ASSEMBLY_DETAILS } from './vis_defines';
+import {
+    SearchBatchDownloadController,
+    BatchDownloadActuator,
+} from './batch_download';
+import { svgIcon } from '../libs/svg-icons';
+import QueryString from '../libs/query_string';
 
+// Generate tabs for available organisms
+const organismTabs = {};
+const organismTerms = ['Homo sapiens', 'Mus musculus'];
+organismTerms.forEach((organismName) => {
+    organismTabs[organismName] = <div id={organismName} className={`organism-button ${organismName.replace(' ', '-')}`}><img src={`/static/img/bodyMap/organisms/${organismName.replace(' ', '-')}.svg`} alt={organismName} /><span>{organismName}</span></div>;
+});
+
+// Not all annotation facets are displayed, these are the ones we want to keep
+const keepFacets = [
+    'assembly', // we actually will not display the assembly facet, but we need it in order to determine which assemblies correspond to results
```

Comment:
Let me try again. There are two "fake" facets at the top which don't function quite like normal facets. They're multi-selects where one selection is required. I'm filtering out most of the facets since we only want a few, but I use the assembly facet data to figure out which assemblies are available and then I use that to generate the fake facet, so I don't want to filter it out. For instance, for Homo sapiens, GRCh38 and hg19 might give results for the selected filters, or maybe only hg19 gives results, so I find that out by looking at the assembly facet, but then I don't actually display that facet.